src/templates/finops-hub/schema/schema_mca_normalized.json

```diff
@@ -0,0 +1,659 @@
+{
+  "additionalColumns": [
```

Comment:
You will need to add a few EA-only columns, like PartNumber, AccountOwnerName, etc.

Reply:
Done